packages/main/tests/robot/test_hubspot.robot

```diff
@@ -0,0 +1,170 @@
+*** Settings ***
+Documentation       API keys and variables must be provided in ./setup.py. These should use a live or
+...                 sandbox Hubspot environment to test the API, there is no mocking function.
+
+Library             RPA.Hubspot
+Library             Collections
+Variables           ./setup.py
```

Comment:
I would call this `testvars.py` or something like that. The `setup.py` has special meaning in the Python packages and I would not like mix that into Robot Framework variables file.

The `testvars.py` can be added into `.gitignore` if that is only blocker related to renaming.

Reply:
Makes perfect sense, this is my first full fledged Python project that I've worked on and I was not aware of `setup.py` significance. I'll update that and switch `retry2` to `tenacity`.